### DISCUSSION TITLE
Nhost has not be initiated!

### DISCUSSION THREAD

**Discussion by mjerjawi** [nhost/nhost#251] [Q&A] [answered]
I'm trying to run Nhost workspace with my Nextjs project and when i run the app i get this 
Error: Nhost has not be initiated! Apollo client can't be created
I tried running the latest Nextjs example from Nhost Github and i ended with the same error can you tell me 
what im doing wrong ?

**Comment by plmercereau** [comment, marked as answer]
Hello Mohammad,
I tried to reproduce the error, with no success. I guess you are trying to use `@nhost/nextjs` with Apollo?
I just started the nextjs example with the following commands :
```sh
pnpm i
cd examples/nextjs
pnpm run dev
```
it works together with a local Nhost instance;
```sh
cd examples/testing-project
nhost -d
```

**Reply by mjerjawi** [reply]
Im trying to run it with nhost workspace, I replaced the localhost url with my nhost backend URL, tbh i did not try to run it with local instance

**Reply by plmercereau** [reply]
I updated [the example](https://github.com/nhost/nhost/tree/main/examples/nextjs) so it is more undestandable.
To use the cloud backend url:
- change `BACKEND_URL` in the `src/helpers/index.ts` file
- change the client URL in the Nhost console so email verification will redirect to localhost: `Users -> Login Settings -> Client login URLs`: `http://localhost:4000`
